civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -5663,11 +5665,12 @@ function setInputDatepickerForm($title = null) {
     '#description' => getCalendarButtonMarkup($idxKey),
     '#date_label_position' => 'within',
     '#attributes' => [
-      'ng-init' => "$idxKey = false; filters.dt$idxKey=filters.date",
-      'placeholder' => '{{filters.placeholderFormat}}',
-      'uib-datepicker-popup' => '{{filters.format}}',
+      'ng-init' => "ilters.$idxKey = false",
```

Comment:
There is a typo. I wonder why it works tho.

Reply:
It did work because `undefined == false`, but it's better to be explicit